### PR TITLE
Create war-on-karma_geoff

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/av/war-on-karma_geoff [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - "https://drive.google.com/file/d/1bZb8_fSP2lXQ2BDoKV70OSg5OI8rP8hX/view?usp=drive_link"
+formats: [mp3]
+course: view
+tags:
+  - karma
+  - time
+  - rebirth
+  -origination
+year: 2006
+month: feb
```
Comment (buddhist-uni) [inline _content/av/war-on-karma_geoff:16]: Good job formatting these 😁

File: _content/av/war-on-karma_geoff [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "The War on Karma"
+authors:
+  - geoff
+external_url: "https://www.dharmaseed.org/talks/5979/"
+drive_links:
+  - "https://drive.google.com/file/d/1bZb8_fSP2lXQ2BDoKV70OSg5OI8rP8hX/view?usp=drive_link"
+formats: [mp3]
+course: view
```
Comment (buddhist-uni) [inline _content/av/war-on-karma_geoff:9]: Let's actually move this to `thought`  Now that I'm relistening to it...

File: _content/av/war-on-karma_geoff [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+tags:
+  - karma
+  - time
+  - rebirth
+  -origination
```
Comment (buddhist-uni) [inline _content/av/war-on-karma_geoff:14]: space before ` origination`

File: _content/av/war-on-karma_geoff [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  - geoff
+external_url: "https://www.dharmaseed.org/talks/5979/"
+drive_links:
+  - "https://drive.google.com/file/d/1bZb8_fSP2lXQ2BDoKV70OSg5OI8rP8hX/view?usp=drive_link"
+formats: [mp3]
+course: view
+tags:
+  - karma
+  - time
+  - rebirth
```
Comment (buddhist-uni) [inline _content/av/war-on-karma_geoff:13]: Let's replace time and rebirth with just `west` perhaps?

**Review by buddhist-uni — COMMENTED** [review]
Filenames must end in `.md`

**Review by buddhist-uni — APPROVED** [review]
Looks good! How'd you manage to rename the file?

**Comment by JJM317** [comment]
I didn't have to rename it actually. When I opened 'edit file', that field
was still editable (is that a word?), so I could just add the .md. I've
forgotten to add that more than a few times - I'll do better (🤞).

kindly,
~Joe
--------------------

**Pull request merged**